sources/packages/web/src/components/generic/StatusBadge.vue

```diff
@@ -37,25 +43,25 @@ export default {
         case GeneralStatusForBadge.Active:
         case GeneralStatusForBadge.ResolvedRestriction:
           // css class for active status
-          badgeClass.value = "status-badge-active";
-          textColor.value = "#333A47";
-          backGroundColor.value = "#FFFFFF";
-          iconColor.value = "#16C92E";
+          badgeClass.value = "status-badge-success";
+          textColor.value = COLOR_BLACK;
+          backGroundColor.value = COLOR_WHITE;
+          iconColor.value = COLOR_BANNER_SUCCESS;
           break;
         case GeneralStatusForBadge.InActive:
           // css class for inactive status
           badgeClass.value = "status-badge-inactive";
-          textColor.value = "#333A47";
-          backGroundColor.value = "#FFFFFF";
+          textColor.value = COLOR_BLACK;
+          backGroundColor.value = COLOR_WHITE;
           iconColor.value = "#333A47";
```

Comment:
maybe you can add `333A47` this too in constants

Reply:
@ann-aot, @guru-aot, @henryoforeh-aot, I talked with @dheepak-aot about having the color constants defined and consumed in this way. I believe that we can try to explore other options once we have the next stable version of the Vuetify ready. Maybe is my lack of knowledge in Vue in general, but I would say that we should try to reuse the global variables for colors already being defined on scss files.

Reply:
@ann-aot @andrewsignori-aot @guru-aot @henryoforeh-aot out of the PR context, we need a sync up to talk and agree about global styling of our components. IMO. Sometime this week itself or next week would be good to have this sync up.